app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save { self.email = email.downcase }
```

Comment:
```suggestion
  before_save { email.downcase! }
```
phân biệt giúp a: downcase vs downcase! nhé

Reply:
downcase và downcase! giống nhau đểu chuyển thành chữ thường hết nhưng downcase! sẽ lưu luôn thay đổi vào email còn downcase thì giá trị email vẫn như thế

Reply:
ready

Reply:
còn gì nữa không e ?

Reply:
downcase! sẽ trả về nil nếu như xâu ký tự không có thay đổi.
Hết ạ!

Reply:
ready